### DISCUSSION TITLE
Why i can't use other proof

### DISCUSSION THREAD

**Discussion by jnjkhjlkjhb8** [community/community#64599] [GitHub Education] [answered]
### Hi everyone,

Hello , Sorry my english not good
I can't change other Proof to verify 
im Taiwan Taoyuan Municipal Tzu-Wen Junior High School student
It say 
`Your application cannot be reviewed until you fix the following:
Your image should be one of the following:
Dated school ID`
And If i use my school ID 
i will get Common verification problems link
i have translate my school ID card and conversion calendar
i don't know the reason for rejected
Someone can help me?

**Comment by arafajs** [comment]
- The ID must be dated and show your current enrollment status.
- The ID must be clear and easy to read.
- The ID must be in English or have an English translation.

**Reply by jnjkhjlkjhb8** [reply]
i have it all

**Reply by jnjkhjlkjhb8** [reply]
wait but my ID only have academic year

**Comment by jnjkhjlkjhb8** [comment, marked as answer]
Ok I just send my transcript and didn't any translate so i pass

**Comment by Daniyal-Aslam** [comment]
**Hello,**

I'm encountering difficulties verifying my school-issued email, [daniyalaslam@students.usindh.edu.pk](mailto:daniyalaslam@students.usindh.edu.pk), for the GitHub Student Developer Pack. I'm a student at the University of Sindh Jamshoro. 

I'm still getting the same error it says (We require applicants to use a school-issued email address to apply.) even after verifying it from my github.

Can anyone please assist in resolving this issue?

**Reply by jnjkhjlkjhb8** [reply]
Maybe you need github support?

**Reply by Daniyal-Aslam** [reply]
How can I approach them?

**Reply by jnjkhjlkjhb8** [reply]
https://support.github.com/contact/education?tags=dotcom-footer%2Chubberfy_education

**Comment by Daniyal-Aslam** [comment]
Hello,

I'm encountering difficulties verifying my school-issued email, [daniyalaslam@students.usindh.edu.pk](mailto:daniyalaslam@students.usindh.edu.pk), for the GitHub Student Developer Pack. I'm a student at the University of Sindh Jamshoro.

I'm still getting the same error it says (we require applicants to use a school-issued email address to apply), even after verifying it from my Github.
